src/jetson/main.py

```diff
@@ -4,6 +4,7 @@
 from typing import List, Set, Dict, Tuple, Optional
 
 import cv2
+from enum import Enum
```

Comment:
Don't see any reference to this. Is it being used?

Reply:
Nope. I'll remove it